lib/web_ui/dev/felt

```diff
@@ -24,27 +24,69 @@ then
 fi
 NINJA_PATH=`which autoninja`
 
+KERNEL_NAME=`uname`
+CPU_NAME=`uname -m`
 ENGINE_SRC_DIR="$(dirname $(dirname $(dirname $(dirname ${FELT_DIR}))))"
 FLUTTER_DIR="${ENGINE_SRC_DIR}/flutter"
+SDK_PREBUILTS_DIR="${FLUTTER_DIR}/prebuilts"
+
+if [ -z "${DART_SDK_DIR}"]
+then
+  if [[ $KERNEL_NAME == *"Darwin"* ]]
+  then
+    TARGET_OS="macos"
+  elif [[ $KERNEL_NAME == *"Linux"* ]]
+  then
+    TARGET_OS="linux"
+  else
+    echo "Unrecognized kernel name: ${KERNEL_NAME}"
+    exit 1
+  fi
+
+  if [[ $CPU_NAME == *"x86_64"* ]]
+  then
+    TARGET_ARCH="x64"
+  elif [[ $CPU_NAME == *"arm64"* ]]
+  then
+    TARGET_ARCH="arm64"
+  else
+    echo "Unrecognized architecture: ${CPU_NAME}"
+    exit 1
+  fi
+
+  PREBUILT_TARGET="${TARGET_OS}-${TARGET_ARCH}"
+  DART_SDK_DIR="${SDK_PREBUILTS_DIR}/${PREBUILT_TARGET}/dart-sdk"
+
+  if [ ! -d "$DART_SDK_DIR" ]
+  then
+    echo "Prebuilt dart sdk for ${PREBUILT_TARGET} not found."
+    echo "Note: You can specify your own path to a built dart sdk with the DART_SDK_DIR environment variable."
+    exit 1
+  fi
+
+  echo "Running felt with prebuilt Dart SDK at ${DART_SDK_DIR}..."
```

Comment:
nit: for a tool that's used multiple times daily I prefer minimizing the amount of printed stuff when things are running along the happy path. The closer we are to `cp` the better. We could have a verbose flag that would output more. Perhaps we should also output more when running on LUCI (I believe there's already a flag for that).